composer.json

```diff
@@ -34,12 +34,12 @@
         }
     ],
     "replace": {
-        "open-telemetry/api": "self.version",
-        "open-telemetry/context": "self.version",
-        "open-telemetry/gen-otlp-protobuf": "self.version",
-        "open-telemetry/sdk": "self.version",
-        "open-telemetry/sdk-contrib": "self.version",
-        "open-telemetry/sem-conv": "self.version"
+        "open-telemetry/api": "^1.0",
+        "open-telemetry/context": "^1.0",
+        "open-telemetry/gen-otlp-protobuf": "^1.0",
+        "open-telemetry/sdk": "^1.0",
+        "open-telemetry/sdk-contrib": "^1.0",
+        "open-telemetry/sem-conv": "^1.0"
```

Comment:
Should probably be exact versions, we don't want `open-telemetry/opentelemetry` to replace future versions of the packages.

Reply:
Yes, I think you're right. We need to work out what `open-telemetry/opentelemetry` is. Since we no longer tag&split the monorepo to a bunch of packages at the same version, I suspect the meta-package only currently works with `dev-main`.
Setting up conflicts in composer between `open-telemetry/opentelemetry` and the other core packages might help this.

Reply:
I set these back to `self.version`. Also, apparently `replace` creates an implicit `conflicts` on the same name, so is not necessary.